### PR TITLE
Blurb about tsconfig.json in 1.8

### PR DESCRIPTION
**Pull request by paulvanbrenk**

### CODE REVIEW

**Comment by johnnyreilly** [comment]
Awesomeness @paulvanbrenk

**Comment by mhegazy** [comment]
Can we add a note on project property pages support and that it is blocked if you have at least one tsconfig file there.

File: What's-new-in-TypeScript.md [hunk 1]
```diff
@@ -485,6 +485,22 @@ As an example, we can pipe our emitted JavaScript into a pretty printer like [pr
 tsc foo.ts --outFile /dev/stdout | pretty-js
 ```
 
+## Improved support for tsconfig.json in Visual Studio 2015
+
+TypeScript 1.8 allows `tsconfig.json` files in all project types, this includes ASP.NET v4, Console Application, and the "Html Application with
```
Comment (sandersn) [inline What's-new-in-TypeScript.md]: all project types**. This** includes

File: What's-new-in-TypeScript.md [hunk 2]
```diff
@@ -485,6 +485,22 @@ As an example, we can pipe our emitted JavaScript into a pretty printer like [pr
 tsc foo.ts --outFile /dev/stdout | pretty-js
 ```
 
+## Improved support for tsconfig.json in Visual Studio 2015
+
+TypeScript 1.8 allows `tsconfig.json` files in all project types, this includes ASP.NET v4, Console Application, and the "Html Application with 
+TypeScript". Further you are no longer limited to a single `tsconfig.json` file, but you can add multiple, and each will be build as part of the project.
```
Comment (sandersn) [inline What's-new-in-TypeScript.md]: each will be **built** as part of the project.

**Comment by paulvanbrenk** [comment]
@mhegazy line 496

File: What's-new-in-TypeScript.md [hunk 3]
```diff
@@ -485,6 +485,22 @@ As an example, we can pipe our emitted JavaScript into a pretty printer like [pr
...
+This allows you to separate the configuration for different parts of your application, without having to use multiple different projects.
```
Comment (sandersn) [inline What's-new-in-TypeScript.md]: Drop 'different' to say "without having to use multiple projects".

**Comment by sandersn** [comment]
:+1:.

**Comment by mhegazy** [comment]
:+1:

File: What's-new-in-TypeScript.md [hunk 4]
```diff
@@ -485,6 +485,22 @@ As an example, we can pipe our emitted JavaScript into a pretty printer like [pr
 tsc foo.ts --outFile /dev/stdout | pretty-js
 ```
 
+## Improved support for tsconfig.json in Visual Studio 2015
+
+TypeScript 1.8 allows `tsconfig.json` files in all project types. This includes ASP.NET v4, Console Application, and the "Html Application with 
+TypeScript". Further, you are no longer limited to a single `tsconfig.json` file but can add multiple, and each will be built as part of the project.
```
Comment (DanielRosenwasser) [inline What's-new-in-TypeScript.md]: TypeScript project types."

File: What's-new-in-TypeScript.md [hunk 5]
```diff
@@ -485,6 +485,22 @@ As an example, we can pipe our emitted JavaScript into a pretty printer like [pr
 tsc foo.ts --outFile /dev/stdout | pretty-js
 ```
 
+## Improved support for tsconfig.json in Visual Studio 2015
+
+TypeScript 1.8 allows `tsconfig.json` files in all project types. This includes ASP.NET v4, Console Application, and the "Html Application with
```
Comment (DanielRosenwasser) [inline What's-new-in-TypeScript.md]: Put these project types in italics, bold, or monospace text.

Comment (DanielRosenwasser) [inline What's-new-in-TypeScript.md]: Each sentence should be on its own line.

File: What's-new-in-TypeScript.md [hunk 6]
```diff
@@ -485,6 +485,22 @@ As an example, we can pipe our emitted JavaScript into a pretty printer like [pr
...
+TypeScript". Further, you are no longer limited to a single `tsconfig.json` file but can add multiple, and each will be built as part of the project.
+This allows you to separate the configuration for different parts of your application without having to use multiple different projects.
+
+![Showing off tsconfig.json in Visual Studio](https://raw.githubusercontent.com/wiki/Microsoft/TypeScript/images/new-in-typescript/tsconfig-in-vs.png)
+
+We also disable the project properties page when you add a `tsconfig.json` file. This means that all configuration changes have to be made in the 
+`tsconfig.json` file itself.
+
+### A couple of limitations:
+* If you add a `tsconfig.json` file, TypeScript files that are not considered part of that context are not compiled.
```
Comment (DanielRosenwasser) [inline What's-new-in-TypeScript.md]: Add a space after the header

Comment (paulvanbrenk) [inline What's-new-in-TypeScript.md]: there already is onw

**Pull request merged**